.github/workflows/docker-develop.yml

```diff
@@ -6,6 +6,9 @@ on:
       - develop
     paths-ignore:
       - 'docs/**'
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
```

Comment:
Why is that added? Was probably just for testing, no?

Reply:
Sorry, debug code. I will clean this up